images/hub/jupyterhub_config.py

```diff
@@ -104,7 +105,7 @@
 
 init_containers = get_config('singleuser.init-containers')
 if init_containers:
-    c.KubeSpawner.singleuser_init_containers = init_containers
+    c.KubeSpawner.singleuser_init_containers += init_containers
```

Comment:
Does that work on traitlets ?

Reply:
`list_trait.extend(other_list)` explicitly works. If `+=` calls extend, this is fine. I'm not sure if that's explicit/official, though so might be reliant on an unguaranteed implementation detail. Switching this to .extend would make me more comfortable.